### PR TITLE
Reverts bimex nerf and soul removal, inscreases bimex price

### PR DESCRIPTION
**Pull request by Blundir**
# About the pull request

Reverts this https://github.com/cmss13-devs/cmss13/pull/7344

Bimex price increased from 2 to 5

# Explain why it's good for the game

Just straightforward butchering soul and lore items in favor of making mp job easier, isn't a good idea

# Changelog
:cl:
add: returned bimex description and flash protection, but price of it inscread from 2 to 5 pref points
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/glasses.dm
Comment (Nanu308) [inline code/modules/clothing/glasses/glasses.dm:1]: Le better solution

File: code/modules/client/preferences_gear.dm
Comment (Nanu308) [inline code/modules/client/preferences_gear.dm:1]: Le better solution

**Review by Nanu308 — APPROVED** [review]

**Comment by efzapa** [comment]
> in favor of making mp job easier

implying that a mechanic that has no use other than making griefing easier is a core mechanic that needs to be preserved because "muh soul"

PRs like these make me wish I could magdump people with rubbershot

**Comment by chefoxara** [comment]
marine buff secured

**Pull request merged**